Start time: 04:27
dataset: sift10M
N: 10000000
dataset file: /mnt/data/mocheng/dataset/sift10m/sift10m.fvecs
query file: /mnt/data/mocheng/dataset/sift10m/sift10m_query.fvecs
attr file: /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/attr_sel_2_100000_random.json
query range file: /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/qrangesel_6_2_100000_random_6.json
ground truth file: /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/sif_gt_sel_6_2_100000_random_6_100.json
K: 100
M: 40
ef_construction: 1000
ef_search: 35
distribution: sel_2_100000_random
collection name: collection_hnsw_sift10M_sel_2_100000_random_efc1000
mode: query
d: 128
collection  collection_hnsw_sift10M_sel_2_100000_random_efc1000  exists
collection size: data: ["{'count(*)': 10000000}"]
query d: 128
get query szie: 10000
get range cnt: 10000
ngt= 10000  nq= 10000  k= 100
2025-07-13 19:27:11,423 [ERROR][handler]: RPC error: [search], <MilvusException: (code=1100, message=failed to create query plan: cannot parse expression: label_attr = 6 and 15386 <= range_attr <= 98713, error: line 1:11 token recognition error at: '= ': invalid parameter)>, <Time:{'RPC start': '2025-07-13 19:27:11.329489', 'RPC error': '2025-07-13 19:27:11.423148'}> (decorators.py:140)
2025-07-13 19:27:11,423 [ERROR][search]: Failed to search collection: collection_hnsw_sift10M_sel_2_100000_random_efc1000 (milvus_client.py:415)
Traceback (most recent call last):
  File "utils/pymilvus_hnsw_arbi.py", line 246, in <module>
    res = client.search(
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/milvus_client/milvus_client.py", line 416, in search
    raise ex from ex
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/milvus_client/milvus_client.py", line 401, in search
    res = conn.search(
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/decorators.py", line 141, in handler
    raise e from e
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/decorators.py", line 137, in handler
    return func(*args, **kwargs)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/decorators.py", line 176, in handler
    return func(self, *args, **kwargs)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/decorators.py", line 116, in handler
    raise e from e
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/decorators.py", line 86, in handler
    return func(*args, **kwargs)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 896, in search
    return self._execute_search(request, timeout, round_decimal=round_decimal, **kwargs)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 837, in _execute_search
    raise e from e
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 826, in _execute_search
    check_status(response.status)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/client/utils.py", line 64, in check_status
    raise MilvusException(status.code, status.reason, status.error_code)
pymilvus.exceptions.MilvusException: <MilvusException: (code=1100, message=failed to create query plan: cannot parse expression: label_attr = 6 and 15386 <= range_attr <= 98713, error: line 1:11 token recognition error at: '= ': invalid parameter)>
Command exited with non-zero status 1
	Command being timed: "python -u utils/pymilvus_hnsw_arbi.py sift10M 10000000 /mnt/data/mocheng/dataset/sift10m/sift10m.fvecs /mnt/data/mocheng/dataset/sift10m/sift10m_query.fvecs /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/attr_sel_2_100000_random.json /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/qrangesel_6_2_100000_random_6.json /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/sif_gt_sel_6_2_100000_random_6_100.json 100 40 1000 35 sel_2_100000_random collection_hnsw_sift10M_sel_2_100000_random_efc1000 query 128"
	User time (seconds): 3.34
	System time (seconds): 5.55
	Percent of CPU this job got: 123%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:07.18
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 130664
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 39833
	Voluntary context switches: 992
	Involuntary context switches: 102564
	Swaps: 0
	File system inputs: 0
	File system outputs: 8
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 1
Start time: 04:28
dataset: sift10M
N: 10000000
dataset file: /mnt/data/mocheng/dataset/sift10m/sift10m.fvecs
query file: /mnt/data/mocheng/dataset/sift10m/sift10m_query.fvecs
attr file: /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/attr_sel_2_100000_random.json
query range file: /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/qrangesel_6_2_100000_random_6.json
ground truth file: /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/sif_gt_sel_6_2_100000_random_6_100.json
K: 100
M: 40
ef_construction: 1000
ef_search: 35
distribution: sel_2_100000_random
collection name: collection_hnsw_sift10M_sel_2_100000_random_efc1000
mode: query
d: 128
collection  collection_hnsw_sift10M_sel_2_100000_random_efc1000  exists
collection size: data: ["{'count(*)': 10000000}"]
query d: 128
get query szie: 10000
get range cnt: 10000
ngt= 10000  nq= 10000  k= 100
2025-07-13 19:28:52,681 [ERROR][handler]: RPC error: [search], <MilvusException: (code=65535, message=fail to search on QueryNode 8: worker(8) query failed: invalid format string)>, <Time:{'RPC start': '2025-07-13 19:28:51.949191', 'RPC error': '2025-07-13 19:28:52.680955'}> (decorators.py:140)
2025-07-13 19:28:52,681 [ERROR][search]: Failed to search collection: collection_hnsw_sift10M_sel_2_100000_random_efc1000 (milvus_client.py:415)
Traceback (most recent call last):
  File "utils/pymilvus_hnsw_arbi.py", line 246, in <module>
    res = client.search(
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/milvus_client/milvus_client.py", line 416, in search
    raise ex from ex
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/milvus_client/milvus_client.py", line 401, in search
    res = conn.search(
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/decorators.py", line 141, in handler
    raise e from e
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/decorators.py", line 137, in handler
    return func(*args, **kwargs)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/decorators.py", line 176, in handler
    return func(self, *args, **kwargs)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/decorators.py", line 116, in handler
    raise e from e
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/decorators.py", line 86, in handler
    return func(*args, **kwargs)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 896, in search
    return self._execute_search(request, timeout, round_decimal=round_decimal, **kwargs)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 837, in _execute_search
    raise e from e
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 826, in _execute_search
    check_status(response.status)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/client/utils.py", line 64, in check_status
    raise MilvusException(status.code, status.reason, status.error_code)
pymilvus.exceptions.MilvusException: <MilvusException: (code=65535, message=fail to search on QueryNode 8: worker(8) query failed: invalid format string)>
Command exited with non-zero status 1
	Command being timed: "python -u utils/pymilvus_hnsw_arbi.py sift10M 10000000 /mnt/data/mocheng/dataset/sift10m/sift10m.fvecs /mnt/data/mocheng/dataset/sift10m/sift10m_query.fvecs /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/attr_sel_2_100000_random.json /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/qrangesel_6_2_100000_random_6.json /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/sif_gt_sel_6_2_100000_random_6_100.json 100 40 1000 35 sel_2_100000_random collection_hnsw_sift10M_sel_2_100000_random_efc1000 query 128"
	User time (seconds): 2.91
	System time (seconds): 2.79
	Percent of CPU this job got: 369%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:01.54
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 131792
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 1
	Minor (reclaiming a frame) page faults: 40340
	Voluntary context switches: 594
	Involuntary context switches: 685306
	Swaps: 0
	File system inputs: 0
	File system outputs: 8
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 1
Start time: 05:24
dataset: sift10M
N: 10000000
dataset file: /mnt/data/mocheng/dataset/sift10m/sift10m.fvecs
query file: /mnt/data/mocheng/dataset/sift10m/sift10m_query.fvecs
attr file: /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/attr_sel_2_100000_random.json
query range file: /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/qrangesel_6_2_100000_random_6.json
ground truth file: /mnt/data/mocheng/dataset/sift10m/label/sel_2_100000_random/sif_gt_sel_6_2_100000_random_6_10.json
K: 10
M: 40
ef_construction: 1000
ef_search: 35
distribution: sel_2_100000_random
collection name: collection_hnsw_sift10M_sel_2_100000_random_efc1000
mode: query
d: 128
collection  collection_hnsw_sift10M_sel_2_100000_random_efc1000  exists
collection size: data: ["{'count(*)': 10000000}"]
query d: 128
get query szie: 10000
get range cnt: 10000
ngt= 10000  nq= 10000  k= 10
